### PR TITLE
profile: Add new profile page for the Hedgehog organisation

### PR DESCRIPTION
**Pull request by qmonnet**
Add a new README.md to display information about Hedgehog and the Fabric on the public GitHub profile page for Hedgehog.

### CODE REVIEW

**Comment by qmonnet** [comment]
Rendered version here: https://github.com/githedgehog/.github/tree/pr/qmonnet/public-profile-init/profile

Merging will display the profile info here: https://github.com/githedgehog?view_as=public

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+[docs]: https://docs.githedgehog.com
+[sonic]: https://sonicfoundation.dev/
+
+### Hedgehog
+
+Hedgehog is the company behind the Open Networking Fabric. For more information
+about us, consult [our website][hedgehog] or take a look at [our blog][blog].
+You can also follow us on [Twitter/X][x] or [LinkedIn][linkedin].
+
```
Comment (mrbojangles3) [inline profile/README.md:40]:
```suggestion

[youtube]: https://www.youtube.com/@hedgehog-sonic
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+documentation][docs].
+
+[docs]: https://docs.githedgehog.com
+[sonic]: https://sonicfoundation.dev/
+
+### Hedgehog
+
+Hedgehog is the company behind the Open Networking Fabric. For more information
+about us, consult [our website][hedgehog] or take a look at [our blog][blog].
+You can also follow us on [Twitter/X][x] or [LinkedIn][linkedin].
```
Comment (mrbojangles3) [inline profile/README.md]:
```suggestion
You can also follow us on [Twitter/X][x], [LinkedIn][linkedin] or [Youtube][youtube].
```

File: profile/README.md (hunk 1 above)
Comment (qmonnet) [reply profile/README.md:40]: Aaaah, I looked for it but couldn't find it, thanks! :heart:

**Comment by qmonnet** [comment]
Fixed, thank you Logan!

**Review by mrbojangles3 — APPROVED** [review]

**Review by Frostman — APPROVED** [review]

**Pull request merged**